docs/guide/configuration/devices-groups.md

```diff
@@ -43,6 +43,17 @@ Every Zigbee Device supports the following list of options.
 **`friendly_name`**  
 Used in the MQTT topic of a device. By default, this is the device ID (e.g. `0x00128d0001d9e1d2`).
 
+::: tip
+You can use the `/` separator in `friendly_name` to _group_ devices.  
```

Comment:
I think "to group devices" is confusing here, this may sound like it creates zigbee groups. Maybe change it to structure devices?